test/napi/src/lib.rs

```diff
@@ -233,6 +234,24 @@ fn main(mut cx: ModuleContext) -> NeonResult<()> {
     cx.export_function("return_js_array_with_string", return_js_array_with_string)?;
     cx.export_function("read_js_array", read_js_array)?;
 
+    cx.export_function("return_js_map", return_js_map)?;
```

Comment:
Can we use the `#[neon::export]` macro for all of the new tests?

Reply:
Yes ! that's much cleaner this way \o/